tinymce-language-selector/LanguagePlugin.js

```diff
@@ -0,0 +1,888 @@
+const BROWSER_DEFAULT = 'BROWSER_DEFAULT';
```

Comment:
No strong opinions where this should live. I think the three options are

1. In this file as it is
2. In the language constant file
3. In a new constants.js file that can keep track of any constants.